openslides/core/static/js/core/base.js

```diff
@@ -701,24 +702,26 @@ angular.module('OpenSlidesApp.core', [
                             {"action": action, "direction": direction}
                     );
                 },
-                getStateForCurrentSlide: function () {
+                getFormForCurrentSlide: function () {
                     var return_dict;
                     angular.forEach(this.elements, function(value, key) {
                         if (value.name == 'agenda/list-of-speakers') {
                             return_dict = {
-                                'state': 'agenda.item.detail',
-                                'param': {id: value.id}
+                                form: $injector.get('TopicForm'),
```

Comment:
This looks wrong. If I have a list of speakers slide, it has not necessarily something to do with topics. Also motion items have list of speakers. And an ItemForm does not exist.